index.html

```diff
@@ -92,23 +92,20 @@
                         </div>
                         <p>Images:</p>
                         <ul>
-                            <li v-if="'qemu' in build.meta.images">
-                                QEMU: <span v-html="getImageUrl(build, 'qemu')"></span>
-                            </li>
-                            <li v-if="'metal' in build.meta.images">
-                                Bare Metal: <span v-html="getImageUrl(build, 'metal')"></span>
+                            <li v-if="'aliyun' in build.meta.images">
+                                Aliyun (Alibaba Cloud): <span v-html="getImageUrl(build, 'aliyun')"></span>
```

Comment:
I felt the "Alibaba Cloud" text was needed to clarify, what do you think?

Reply:
Yup, WFM!